### PR TITLE
feat: Upgrade cozy-client to 33.0.0

### PR DESCRIPTION
**Pull request by Ldoppea**
`cozy-client` as been upgraded to to `33.0.0` to retrieve improved
useCapabilities from cozy/cozy-client#1227

This related cozy-client contains [a Breaking Change on this method](https://github.com/cozy/cozy-client/pull/1227/commits/ea05b01bb12ca608d2f5c9ef2d1ecdb5b55846bf), so I fixed it

### CODE REVIEW

**Review by Crash-- — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "cozy-authentication": "2.8.3",
     "cozy-bar": "7.19.1",
     "cozy-ci": "0.4.1",
-    "cozy-client": "^32.2.8",
+    "cozy-client": "^33.0.0",
```
Comment (Crash--) [inline package.json:119]: @Ldoppea j'ai approved trop tôt. Faut aussi mettre à jour stack-client et pouch-link. repo-doctor n'est pas content sinon ;)

Comment (Ldoppea) [reply package.json:119]: Done here : https://github.com/cozy/cozy-drive/compare/75ffe9d8757a3635264be1001fca4cb8caf61a13..75182823b73087eba81c3248081a7707c46069c4

Mais du coup maintenant c'est bundlemon qui est pas content 😅

**Comment by Ldoppea** [comment]
> @Ldoppea we have a few issues in our test suite: https://app.travis-ci.com/github/cozy/cozy-drive/jobs/580643061#L382

Should be fixed in https://github.com/cozy/cozy-drive/compare/75ffe9d8757a3635264be1001fca4cb8caf61a13..75182823b73087eba81c3248081a7707c46069c4 (waiting for CI)

**Review by Crash-- — APPROVED** [review]
bundlemon is too strict there. Thanks for the upgrade @Ldoppea 👍

**Pull request merged**